main.go

```diff
@@ -239,6 +240,8 @@ func generateEngine() *gin.Engine {
 		profileEditHandlers.ProfileBackgroundSubmitHandler,
 	)
 
+	r.GET("/link-discord", connectionHandlers.LinkDiscordHandler)
```

Comment:
does it have to be a GET for any reason? seems more like a `POST /user/{user_id}/discord-association` or something like that maybe but nbd

Reply:
user id URL is redundant because we're reading the user ID from cookies. happy to change URL to whatever else though. we can make it a POST request but will require some JS as currently it's just an anchor tag